### PR TITLE
Re-styled inactive state, active state and add/edit rsvp

### PR DESCRIPTION
**Pull request by estebanhdzc**
### 🎫 Ticket

[ET-1823]
[ET-1825]
[ET-1836]

### 🗒️ Description

- Re-styled inactive state
- Re-styled active state 
- Re-styled add/edit rsvp

### 🎥 Artifacts 
https://www.loom.com/share/c3b2407067124f94bb48ff9a1d13d990

### ✔️ Checklist
- [x] I've included a changelog entry in the readme.txt file. 
- [x] My code is tested. 
- [x] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[ET-1823]: https://stellarwp.atlassian.net/browse/ET-1823?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ET-1825]: https://stellarwp.atlassian.net/browse/ET-1825?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ET-1836]: https://stellarwp.atlassian.net/browse/ET-1836?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/modules/blocks/rsvp/capacity/styles.pcss [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.tribe-editor__ticket__content-row--capacity {
+	margin-bottom: 20px;
```
Comment (juanfra) [inline src/modules/blocks/rsvp/capacity/styles.pcss:2]: Maybe we could use `--tec-spacer-4`?

File: src/modules/blocks/rsvp/container-header/template.js [hunk 2]
```diff
@@ -102,53 +45,61 @@ const getCapacityLabel = ( capacity ) => {
...
-	onTempTitleChange,
-	tempDescription,
-	tempTitle,
 	title,
 	available,
+	setAddEditOpen,
 } ) => {
+
+	if ( isAddEditOpen ) {
+		return null;
```
Comment (juanfra) [inline src/modules/blocks/rsvp/container-header/template.js:56]: 👌

File: src/modules/blocks/rsvp/container-header/template.js [hunk 3]
```diff
@@ -102,53 +45,61 @@ const getCapacityLabel = ( capacity ) => {
...
+	}
+
+	const leftSide = (
```
Comment (juanfra) [inline src/modules/blocks/rsvp/container-header/template.js:59]: Nit: Not sure if `side` would be too generic, I think we would normally use column, you can maybe double check with Stephen. It's just that I didn't see us using that term for layouts, I think we would normally use columns.

File: src/modules/blocks/rsvp/description/styles.pcss [hunk 4]
```diff
@@ -0,0 +1,12 @@
+.tribe-editor__ticket__content-row--description {
+	margin-bottom: 20px;
```
Comment (juanfra) [inline src/modules/blocks/rsvp/description/styles.pcss:2]: We could maybe use `--tec-spacer-4` here.

File: src/modules/blocks/rsvp/title/styles.pcss [hunk 5]
```diff
@@ -0,0 +1,18 @@
+.tribe-editor__ticket__content-row--title {
+	margin-bottom: 20px;
```
Comment (juanfra) [inline src/modules/blocks/rsvp/title/styles.pcss:2]: Same comment about using `--tec-spacer-4` for this one.

**Review by juanfra — COMMENTED** [review]
Nice work! It's looking great, I have left a few comments.

File: src/modules/blocks/rsvp/capacity/styles.pcss (hunk 1 above)
Comment (estebanhdzc) [reply src/modules/blocks/rsvp/capacity/styles.pcss]: Thanks! I've applied this fix

File: src/modules/blocks/rsvp/container-header/template.js (hunk 3 above)
Comment (estebanhdzc) [reply src/modules/blocks/rsvp/container-header/template.js]: I've renamed this to leftColumn and rightColumn

File: src/modules/blocks/rsvp/title/styles.pcss (hunk 5 above)
Comment (estebanhdzc) [reply src/modules/blocks/rsvp/title/styles.pcss]: Thanks! I've applied this fix

File: src/modules/blocks/rsvp/container-header/style.pcss [hunk 6]
```diff
@@ -6,6 +6,26 @@
...
+	&.tribe-editor__rsvp--add-edit-open {
+		.tribe-editor__rsvp-container .tribe-editor__container-panel__content {
+			display: block;
+		}
+	}
+
+	.tribe-editor__rsvp-container .tribe-editor__container-panel__content {
+		border: 1px solid var(--tec-color-border-secondary);
+		border-bottom: none;
+		background-color: var(--tec-color-background-secondary);
```
Comment (codingmusician) [inline src/modules/blocks/rsvp/container-header/style.pcss:20]:
```suggestion
		background-color: var(--tec-color-background-secondary);
		border: 1px solid var(--tec-color-border-secondary);
		border-bottom: none;
```

File: src/modules/blocks/rsvp/container-header/style.pcss [hunk 7]
```diff
@@ -125,11 +149,12 @@
...
-.tribe-editor__rsvp-container-header__capacity-label {
+.tribe-editor__numeric-label.tribe-editor__rsvp-container-header__capacity-label {
 	display: flex;
-	color: #545D66;
-	padding-top: 15px;
-	line-height: 18px;
+	color: var(--tec-color-text-secondary);
+	font-size: var(--tec-font-size-1);
+	padding-top: var(--tec-spacer-4);
+	line-height: var(--tec-line-height-2);
```
Comment (codingmusician) [inline src/modules/blocks/rsvp/container-header/style.pcss:157]:
```suggestion
	line-height: var(--tec-line-height-2);
	padding-top: var(--tec-spacer-4);
```

File: src/modules/blocks/rsvp/container-header/style.pcss [hunk 8]
```diff
@@ -139,10 +164,10 @@
...
 
 	.tribe-editor__rsvp-container-header__capacity-label & {
-		flex: none;
-		font-weight: bold;
-		font-size: 18px;
-		letter-spacing: 0.05px;
+
+		font-weight: var(--tec-font-weight-bold);
+		font-size: var(--tec-font-size-1);
+		line-height: var(--tec-line-height-2);
```
Comment (codingmusician) [inline src/modules/blocks/rsvp/container-header/style.pcss:170]:
```suggestion
		font-size: var(--tec-font-size-1);
		font-weight: var(--tec-font-weight-bold);
		line-height: var(--tec-line-height-2);
```

File: src/modules/blocks/rsvp/container-header/style.pcss (hunk 6 above)
Comment (estebanhdzc) [reply src/modules/blocks/rsvp/container-header/style.pcss]: Thanks! This suggestion was applied 😃

File: src/modules/blocks/rsvp/container-header/style.pcss (hunk 7 above)
Comment (estebanhdzc) [reply src/modules/blocks/rsvp/container-header/style.pcss]: Thanks! This suggestion was applied 😃

File: src/modules/blocks/rsvp/container-header/style.pcss (hunk 8 above)
Comment (estebanhdzc) [reply src/modules/blocks/rsvp/container-header/style.pcss]: Thanks! This suggestion was applied 😃

File: src/modules/blocks/rsvp/style.pcss [hunk 9]
```diff
@@ -81,16 +86,34 @@
...
+		.tribe-editor__rsvp-actions-rsvp {
+			align-items: center;
+			display: flex;
+			height: 100%;
+
+			.tribe-editor__rsvp-actions-rsvp-create {
+				display: flex;
+				justify-content: space-between;
+				flex-direction: column;
+				height: 100%;
```
Comment (codingmusician) [inline src/modules/blocks/rsvp/style.pcss:107]:
```suggestion
				display: flex;
				flex-direction: column;
				height: 100%;
				justify-content: space-between;
```

File: src/modules/blocks/rsvp/description/template.js [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+			/>
+
+			<Input
+				className="tribe-editor__ticket__description-input"
+				id={ descriptionId }
+				disabled={ isDisabled }
+				type="text"
+				value={ tempDescription }
+				onChange={ onTempDescriptionChange }
+			/>
```
[the remark this answers is not in the document]
Comment (estebanhdzc) [reply src/modules/blocks/rsvp/description/template.js:48]: Awesome, thanks! This is fixed now

File: src/modules/blocks/rsvp/style.pcss (hunk 9 above)
Comment (estebanhdzc) [reply src/modules/blocks/rsvp/style.pcss:107]: Thanks! This is fixed now

**Review by juanfra — APPROVED** [review]
🚢 🏆

**Pull request merged**